presto-main/src/main/java/io/prestosql/execution/StageStats.java

```diff
@@ -210,6 +210,11 @@ public DateTime getSchedulingComplete()
         return schedulingComplete;
     }
 
+    /**
+     * Gets an object with statistics about the splits used by a stage.
```

Comment:
Do you think we should explain what is a "split" here?

Reply:
I do not think so, because there is already a description about it in Presto documentation and it is a pre-requisite to study Presto flow.